content/en/deployment/monitor-deployment.md

```diff
@@ -6,12 +6,13 @@ description: >
   Monitor the status of your deployments and approve or rollback deployments.
 categories: ["Deployment", "Guides"]
 tags: ["Monitor"]
+draft: true
```

Comment:
armory.io links to this page

Reply:
![image](https://github.com/armory/docs-cdaas/assets/26799583/a0a7b09c-6784-4e9f-bcd1-61ca77018d44)

Reply:
Noted, will remove the draft flag for now. outside MVC scope to update the content here. 
Not sure if armory.io should be linking to this page but we can revisit that later.